<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Participants</class>
 <widget class="qMRMLWidget" name="Participants">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>445</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTableWidget" name="participantsTable">
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Surname</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Last Recording</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Other</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>List of participants</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="checkRecordingsButton">
           <property name="text">
            <string>Check recordings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editParticipantButton">
           <property name="text">
            <string>Edit participant</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteParticipantButton">
           <property name="text">
            <string>Delete participant</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="participantSearchText"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Edit participant</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="editParticipantNameText"/>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="saveEditButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="editParticipantSurnameText"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Surname:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="cancelEditButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
